src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1913,13 +1913,13 @@ static SvgStyleGradient* _createRadialGradient(SvgLoaderData* loader, const char
         return nullptr;
     }
     /**
-    * Default values of gradient
+    * Default values of gradient transformed into global percentage
     */
-    grad->radial->cx = 0.5;
-    grad->radial->cy = 0.5;
-    grad->radial->fx = 0.5;
-    grad->radial->fy = 0.5;
-    grad->radial->r = 0.5;
+    grad->radial->cx = 0.5 / loader->svgParse->global.w;
```

Comment:
1. Here it verifies w/h is greater than 0?

2. please append *f  for the float types. => 0.5f;

Reply:
it may be zero and then it's a problem for defaults but also for the values given by the user (there is also division by 0).
I'll think about this 'global' variable...